AustinHarris.JsonRpcTestN/Test.cs

```diff
@@ -1879,6 +1879,13 @@ public void TestNestedReturnType()
             Assert.AreEqual(expected, result.Result);
         }
 
+        [Test]
+        public void InstantiatingAServiceMoreThanOnceDoesNotThrow()
+        {
+            Assert.DoesNotThrow(()=>new CalculatorService());
+            Assert.DoesNotThrow(() => new CalculatorService());
```

Comment:
So the reason it is failing is that two different instances are being bound to the same SessionId. If two instances are desired, then they need different sessionIds